app/src/main/java/uk/gov/onelogin/mainnav/nav/MainNavRoutes.kt

```diff
@@ -1,22 +1,23 @@
-package uk.gov.onelogin.ui.home
+package uk.gov.onelogin.mainnav.nav
 
 import androidx.navigation.NavGraphBuilder
 import androidx.navigation.compose.composable
 import androidx.navigation.navigation
+import uk.gov.onelogin.mainnav.ui.MainNavScreen
 
-object HomeRoutes {
+object MainNavRoutes {
     const val ROOT: String = "/home"
     const val START: String = "$ROOT/start"
 
-    fun NavGraphBuilder.homeFlowRoutes() {
+    fun NavGraphBuilder.mainNavRoutesFlow() {
         navigation(
             route = ROOT,
             startDestination = START
         ) {
             composable(
                 route = START
             ) {
-                HomeScreen()
+                MainNavScreen()
```

Comment:
You don't want to pass the NavController to the `MainNavScreen`?

Reply:
We need to look into this, how we do the nested navigation, think something to come back for another day